src/webui/webapplication.cpp

```diff
@@ -367,6 +367,8 @@ void WebApplication::action_command_download()
     CHECK_URI(0);
     QString urls = request().posts["urls"];
     QStringList list = urls.split('\n');
+    bool skipChecking = request().posts["skip_checking"].trimmed() == "true";
+    bool addPaused = request().posts["paused"].trimmed() == "true";
```

Comment:
Values can be case sensitive for example when we rename a torrent. 

IMO when we send boolean values in a request params, we must agree that only the string value "true" meaning boolean true (not "True", "1" or something else).

Also is ```.trimmed()``` really nead?